### PR TITLE
Changed how the font.json file was loaded to reduce the final webpack…

### PR DESCRIPTION
**Pull request by lemccarthy**
… bundle size

### CODE REVIEW

File: src/shared/lib/svgToPdfDownload.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import svg2pdf from "svg2pdf.js";
 import {jsPDF} from "jspdf-yworks";
+import font from "font.json";
```
Comment (alisman) [inline src/shared/lib/svgToPdfDownload.ts]: lets change the name of this file to something more descriptive.  like pdf-font.   or perhaps the name of the actual font.

File: vendor-bundles.webpack.config.js [hunk 2]
```diff
@@ -37,6 +40,10 @@ let config = {
     devtool : 'source-map'
 };
 
+config.resolve = {
+    modules: [path.join(__dirname, "src", "common"), "node_modules"]
```
Comment (alisman) [inline vendor-bundles.webpack.config.js]: i doubt you need __dirname here.   if we needed it, wouldn't it be necessary for node_modules as well?

File: src/shared/lib/svgToPdfDownload.ts (hunk 1 above)
Comment (lemccarthy) [reply src/shared/lib/svgToPdfDownload.ts]: Got it, changed the name.

File: vendor-bundles.webpack.config.js (hunk 2 above)
Comment (lemccarthy) [reply vendor-bundles.webpack.config.js]: Ok, removed

File: src/shared/lib/svgToPdfDownload.ts [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import svg2pdf from "svg2pdf.js";
 import {jsPDF} from "jspdf-yworks";
+import font from "FreeSans.json";
 import _ from "lodash";
 
+console.log(font);
```
Comment (alisman) [inline src/shared/lib/svgToPdfDownload.ts]: kill console

**Comment by alisman** [comment]
@jjgao merging this with 5 unrelated e2e tests failing

**Pull request merged**